spec/components/deal.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Deal from '../../client/src/components/deal';
+
+describe('Deals Component', () => {
+  const data = {
+    id: 7,
+    title: 'quas nemo et',
+    msrp: '658.99',
+    salePrice: '619.46',
+    extraPrice: '576.10',
+    bought: 140,
+    allowed: 340,
+    soldOut: true,
+    salePercent: 6,
+    extraPercent: 6,
+    packageId: 4,
+    createdAt: '2020-01-28T22:22:24.000Z',
+    updatedAt: '2020-01-28T22:22:24.000Z',
+  };
```

Comment:
Not sure this data is needed for this specific test. I believe that shallow rendering without any deal prop would result the same.

Reply:
Thank you for taking a look at this. I put in the data in order to write better test in the future. You are right for this specific test I didn't have to put this in